naptime-graphql/src/test/scala/org/coursera/naptime/ari/graphql/schema/NaptimeUnionFieldTest.scala

```diff
@@ -108,4 +108,38 @@ class NaptimeUnionFieldTest extends AssertionsForJUnit with MockitoSugar {
     assert(field.fieldType.toString === expectedField.toString)
   }
 
+  @Test
+  def build_ShorthandTypedDefinitionUnion() = {
```

Comment:
sorry to be obnoxious, but can you modify this test / add one more test that ensures that shorthand syntax doesn't work if the union member is in a different namespace than the parent? (i left a comment above about also modifying the code to support this -- i didn't catch it in the first run, sorry!)